### DISCUSSION TITLE
My angular datatable is working fine but all click methods are working only  in desktop view but in responsive view for mobile and desktop click function is not working in angular.

### DISCUSSION THREAD

**Discussion by indranilc98** [l-lin/angular-datatables#1573] [Q&A]
responsive: {
        details: {
            renderer: Responsive.renderer.listHiddenNodes()
            listHiddenNodes()
        }
    after this command, in responsive view is working but the data is not loading properly. First row data is showing for every row.

**Comment by shanmukhateja** [comment]
Hi,

Could you provide a reproducible GitHub repo or StackBlitz link to investigate this? Your "code snippet" does not make sense.
